sycl/include/CL/sycl/kernel_bundle.hpp

```diff
@@ -0,0 +1,572 @@
+//==------- kernel_bundle.hpp - SYCL kernel_bundle and free functions ------==//
+//
```

Comment:
Do not add a test in this patch since not sure if it makes a lot of seance to prepare "compile-only" test.